src/components/pages/Home/RenderHomePage.js

```diff
@@ -1,11 +1,15 @@
 import React from 'react';
 import { Link } from 'react-router-dom';
 import { Button } from '../../common';
+import { MenuBar } from '../MenuBar/index';
+import { LoadingComponent } from '../../common';
 
 function RenderHomePage(props) {
+  console.log('homeprops', props);
   const { userInfo, authService } = props;
   return (
     <div>
+      <MenuBar authService={authService} />
```

Comment:
We should move this outside of the routes so that it shows up regardless of which page you're on